src/i18n/locales/en/translation.json

```diff
@@ -743,11 +743,16 @@
       "content": "This website asks you to sign:",
       "allow_sign": "Allow {{host}} to sign:",
       "allow_sign_event": "Allow {{host}} to sign a {{kind}}:",
+      "allow_sign_event_contacts": "Allow {{host}} to sign an update to contacts list",
       "view_details": "View details",
       "hide_details": "Hide details",
       "no_content": "(No content)",
       "block_and_ignore": "Block and ignore {{host}}",
       "block_added": "Added {{host}} to the blocklist, please reload the website.",
+      "reactions": {
+        "like": "🤙 Like"
```

Comment:
I don't think we need to add a translation for each emoji. The emoji should be self-explanatory